pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -245,6 +245,7 @@ t_ndarray array_slicing(t_ndarray arr, int n, ...)
     va_list  va;
     t_slice slice;
     int32_t start = 0;
+    int32_t offset = 0;

```

Comment:
Can you please explain what `offset` represents?

Reply:
`offset` is the position of the first slice with the step bigger than 1, it just gives an idea of what dimension from which the copying begins.

Reply:
Good job for avoiding the double allocation!